modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -379,7 +395,42 @@ sda.enkf.multisite <- function(settings,
       if (t>1){
         #for next time step split the met if model requires
         #-Splitting the input for the models that they don't care about the start and end time of simulations and they run as long as their met file.
-        inputs.split <- metSplit(conf.settings, inputs, settings, model, no_split = FALSE, obs.times, t, nens, restart_flag = FALSE, my.split_inputs)
+        #set start and end date for splitting met
+        start.time = obs.times[t - 1] #always start timestep before
+        
+        if(restart_flag){
+          stop.time = settings$run$site$met.end
+        }else{
+          stop.time = obs.times[t]
+        }
+        
+        
+        #-Splitting the input for the models that they don't care about the start and end time of simulations and they run as long as their met file.
+        inputs.split <- 
+          furrr::future_pmap(list(conf.settings %>% `class<-`(c("list")), inputs, model), function(settings, inputs, model) {
+            # Loading the model package - this is required bc of the furrr
+            library(paste0("PEcAn.",model), character.only = TRUE)
```

Comment:
We shouldn't have to load library's like this. But if you do, load it once, before the pmap, not repeatedly.

Reply:
Has removed this block of code.